src/pages/[locale]/work/[slug].astro

```diff
@@ -75,15 +75,11 @@ const { data: casePage } = await loadQuery<WorkWithReferences>({
   slug={slug}
   siteConfig={siteConfig}
   translations={casePage?._translations}
-  routeMap="cases"
+  routeMap="work"
   noIndex={casePage?.metadata?.noIndex}
   seoImage={casePage?.metadata?.image as Image}
 >
-  <HeroHeader
-    headline={casePage?.intro?.title}
-    text={casePage?.intro?.intro}
-    backgroundImage={casePage?.image?.asset?._id}
-  >
+  <HeroHeader headline={casePage?.intro?.title} text={casePage?.intro?.intro}>
```

Comment:
are you sure we don't want background images here? https://dreamy-trifle-7f42a6.netlify.app/fuse/en/work/argenta/ seems to have it

Reply:
my bad, I removed backgroundImage for testing and forgot to add it back.